### ISSUE TITLE
Age dates not used

### ISSUE THREAD

**Issue by bethany-j-allen** [paleobiodb/PBDB_changelog#55]
In the Stratigraphy tab, enterers are required to give an oldest time interval. They can also give a youngest time interval, to specify an age range. There are additionally options to give a direct date, or range of dates, based on a variety of methods. The age actually allocated to the collection, both on the website and in downloads, appears to be wholly determined by the intervals given, and the date information is not used anywhere. Ideally it would make sense to use the dates if provided, as these are likely to be more accurate.
<img width="832" alt="Screenshot 2023-07-17 at 16 05 15" src="https://github.com/paleobiodb/bug_reports/assets/39991015/ac3d2306-7b34-4bcb-b491-9dfba5e190eb">
<img width="808" alt="Screenshot 2023-07-17 at 16 04 54" src="https://github.com/paleobiodb/bug_reports/assets/39991015/7304e9e5-e495-4dde-98b1-c5e0767bae96">
<img width="1430" alt="Screenshot 2023-07-17 at 16 05 45" src="https://github.com/paleobiodb/bug_reports/assets/39991015/8427c56c-26ef-48e2-b144-2b89e259d865">

**Comment by mmcclenn** [comment]
You are right, this is a deficiency in the API output. I will check with some of the other power users, and see if they are okay with reporting the explicit maximum and minimum dates for any collection which specifies them.
